backend/infrahub/events/node_action.py

```diff
@@ -33,13 +32,4 @@ def get_payload(self) -> dict[str, Any]:
         return self.data
 
     def get_messages(self) -> list[InfrahubMessage]:
-        return [
-            EventNodeMutated(
```

Comment:
I think we should keep this event for now.
we need to do some changes but completely removing it now means we'll need to redo everything shortly after

Reply:
The main reason for opening this PR was actually to remove this as it generates a lot of extra noise and slows things down when a there's a high number of nodes that are mutated. I'm not sure I see the benefits of keeping it and what we'd need to redo later? (i.e. specifically related to this message)

Reply:
I've added back the message but removed the flow from it, so at the moment it doesn't do anything aside from writing a debug log entry. But it's there for future use if we want it.